FirebaseTestProject/Views/ProfileView.swift

```diff
@@ -40,9 +66,29 @@ struct ProfileView: View {
             .disabled(signOutTask.isInProgress)
             Spacer()
         }
+        .alert("Cannot Update Profile Photo", isPresented: $updateImageTask.isError, presenting: updateImageTask.error, actions: { _ in }) { error in
+            Text(error.localizedDescription)
+        }
         .alert("Cannot Sign Out", isPresented: $signOutTask.isError, presenting: signOutTask.error, actions: { _ in }) { error in
```

Comment:
Multiple alert modifiers is an interesting pattern.  Is there a way to consolidate these, or is this an established practice?

Reply:
This isn’t an established practice AFAIK, so consolidating these would be a good idea.

The simplest way to consolidate these would be to create one catch-all alert. This would mean the alert title would have to the same for both profile photo and authentication errors (unless we’re willing to use the [now-deprecated alert API](https://developer.apple.com/documentation/swiftui/view/alert(ispresented:content:))).

The alternative, which I think is preferable, would be the [new `alert(isPresented:error:actions:message:)` modifier](https://developer.apple.com/documentation/swiftui/form/alert(ispresented:error:actions:message:)). To use that modifier, we’d need to create custom error types that conform to `LocalizedError` (the `errorDescription` property becomes the alert title).

Please let me know what you think!

Reply:
This does seem like a better approach, yeah.  Let's see what this would look like.  A single alert title is fine I think.

Reply:
I tried consolidating the alerts. Does this work?